frontend/test/app/components/NotFoundPage-test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+import NotFoundPage from '../../../src/app/components/NotFoundPage';
+
+describe('app/components/NotFoundPage', () => {
+  it('should render notFoundHeader string', () => {
+    expect(shallow(<NotFoundPage />)
+      // HACK: .find('[data-l10n-id="errorMessage"]') seems not to work
```

Comment:
Seems like attribute names with dashes make enzyme unhappy. Maybe better to actually add some descriptive classes to these elements as test handles?